### PR TITLE
Adds tc-health-client to CDN-in-a-Box

### PR DESCRIPTION
**Pull request by fabikd**
Adds `tc-health-client` to CDN-in-a-Box, fix to show the FQDN instead of hostname and enabled logging sending it to the default log file instead of `/dev/null`.

<hr/>

## Which Traffic Control components are affected by this PR?
- Traffic Control Health Client (tc-health-client)
- CDN in a Box

## What is the best way to verify this PR?

Build and run cdn-in-a-box, once up and running, using docker desktop, navigate to terminal tab of an edge or mid. cd into `/var/log/trafficcontrol` and run `tail -f tc-health-client.log`. click on the open a terminal link on upper right side and cd into `/usr/bin` and run `./tc-health-client`. wait for the dispersion time to pass and then logs will start in the window where the `tail` command was ran.

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [ ] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Comment by ocket8888** [comment]
Can you please add a description to this PR? As it stands, I have no idea what the intended change is.

**Comment by fabikd** [comment]
sure!! (:  it was under the pr submission checklist. But it was hard to see, my fault. Here it is:

Adds tc-health-client to cdn-in-a-box, fix to show the fqdn instead of hostname and enabled logging sending it to the default log file instead of /dev/null.

**Review by jpappa200 — CHANGES_REQUESTED** [review]
Looks good and works by following directions in this PR. 
Would like to see the `/etc/to-creds` file automatically populated so all that is necessary is to start the health client.
Once I had it running though I did see that it couldn't find a trafficmonitor. `ERROR: tmhealthservice.go:109: 2023-05-30T17:26:36.760257802Z: error in TrafficMonitor polling: finding a trafficmonitor: there are no available traffic monitors`

**Comment by fabikd** [comment]
Hi Joe!!! I just pushed a fix for the error we were seeing in the cdn-in-a-box finding the traffic_monitor. This will look for it with an underscore`RASCAL_Traffic_Monitor` instead of `RASCAL-Traffic_Monitor` and I populated the `/etc/to-creds` with the cdn-in-a-box creds!

**Comment by mitchell852** [comment]
@fabikd would you mind making the PR title more descriptive? maybe "Adds tc-health-client to CDN-in-a-Box"

**Comment by fabikd** [comment]
pr name updated! (:

File: infrastructure/cdn-in-a-box/health/tc-health-client.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
-    "cdn-name": "over-the-top",
+    "cdn-name": "mycdn",
```
Comment (jpappa200) [inline infrastructure/cdn-in-a-box/health/tc-health-client.json:2]: This should be `CDN-in-a-Box`

**Review by jpappa200 — CHANGES_REQUESTED** [review]

File: infrastructure/cdn-in-a-box/health/tc-health-client.json (hunk 1 above)
Comment (fabikd) [reply infrastructure/cdn-in-a-box/health/tc-health-client.json]: Done!! (:

**Review by jpappa200 — APPROVED** [review]
👍

**Pull request merged**